shoop/admin/views/dashboard.py

```diff
@@ -22,3 +23,7 @@ def get_context_data(self, **kwargs):
             blocks.extend(module.get_dashboard_blocks(request=self.request))
         context["activity"] = get_activity(request=self.request)
         return context
+
+    def get(self, request, *args, **kwargs):
+        send_telemetry(request)
+        return super(DashboardView, self).get(request, *args, **kwargs)
```

Comment:
Still, it makes the code less readable. At least the function should be named "send_if_supposed" or something like that then...